llvm/lib/MC/MCMachOStreamer.cpp

```diff
@@ -203,11 +203,11 @@ void MCMachOStreamer::emitDataRegion(MachO::DataRegionType Kind) {
   MCSymbol *Start = getContext().createTempSymbol();
   emitLabel(Start);
   // Record the region for the object writer to use.
-  getWriter().getDataRegions().push_back({Kind, Start, nullptr});
```

Comment:
Nit: I will just remove or comment out `getWriter()` method with an explanation for why we should avoid using in downstream. Otherwise LGTM.

Reply:
Yes, that was supposed to be done, but I forgot 😅